doc/refman/RefMan-ltac.tex

```diff
@@ -600,6 +613,11 @@ \section{Semantics}
 
 \ErrMsg \errindex{Cannot solve the goal}
 
+\variant {\tt solve {\tacexpr}}
+
+This is an Ltac alias that gives a primitive access to the {\tt solve} tactical.
+See the {\tt first} tactical for more information.
```

Comment:
The space immediately after `first}` is the unicode non breaking space character.
If you get the raw log from travis the first character after \u8: in
>! Package inputenc Error: Unicode char \u8:  not set up for use with LaTeX. (file RefMan-ltac.v.tex)

is the same, so I think this is why the build is broken.